### ISSUE TITLE
IntelliSense for 'http' Node.js module does not work on macOS

### ISSUE THREAD

**Issue by mcameriere** [Microsoft/vscode#33661]
- VSCode Version: 1.15.1
- OS Version: macOS Sierra 10.12.6

Steps to Reproduce:

1. Open Terminal
2. mkdir intellisense-demo
3. cd intellisense-demo
4. code .
5. Create a new index.js file with the following content
**// index.js
var http = require('http');
http.c**
6. launch suggestions ctrl+space

=> No suggestions

Reproduces without extensions: Yes

However, IntelliSense works for client-side javascript like "console.log" or "document.getElementById" and the likes...

**Comment by mjbvz** [comment]
Hmm, looks like we are not downloading the typing file for `node` or `http`. These typings files power VSCode's JavaScript language support ([more info](https://code.visualstudio.com/Docs/languages/javascript#_automatic-type-acquisition))

To investigate, can you please:

1. Set `"typescript.tsserver.log": "verbose"` in your user settings
2. Restart VSCode and open the file again
3. Wait 10 seconds or so
4. Run the `TypeScript: Open TS Server Log` command in VSCode
4. This should open a folder with a file called `ti-xxx.log` That file has info about typings installation

Can you please share this log file or look through it to see if there are any errors? Let me know if you have any questions about collecting this

**Comment by mcameriere** [comment]
Hi! Many thanks for your quick answer.

Thanks to the ti-xxx.log I found the issue !

It was caused by the npm registry I used.
I changed the npm registry to a local proxy we use internally at our company.
After changing the npm registry with the following command:

npm config set registry https://registry.npmjs.org/

and restating VS Code. IntelliSense works for built-in Node modules. Great!

Thank you!
